packages/iov-bns/src/encode.ts

```diff
@@ -62,6 +62,15 @@ function encodeString(data: string | undefined): string | null {
   return data || null;
 }
 
+const maxMemoLength = 128;
+
+function encodeMemo(data: string | undefined): string | null {
+  if (data && Encoding.toUtf8(data).length > maxMemoLength) {
+    throw new Error("Invalid memo length: maximum 128 bytes");
```

Comment:
Can we either use the constant in the error message or remove the number from the message?